root/report/ArtistsThatMayBeGroups.js

```diff
@@ -24,10 +24,12 @@ component ArtistsThatMayBeGroups(...{
       description={l(
         `This report lists artists that have their type set to other
          than Group (or a subtype of Group) but may be a group,
-         because they have other artists listed as members. If you find
+         because they have other artists listed as members or use other
+         relationships that are generally meant for groups only. If you find
```

Comment:
It would be informative for the non-developers to know all of the relationship types being currently checked:

```suggestion
          relationships that are generally meant for groups only, at least in
          one direction (for its “artistic director”, “conductor”, “composer
          in residence”, “subgroup”, or when “renamed”). If you find
```